File: sandbox/node/src/TeamInvitesDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TeamApi();

const emailAddress = undefined;

apiCaller.teamInvites(
    emailAddress,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Team#teamInvites:");
  console.log(error.body);
});

File: sandbox/dotnet/src/Dropbox.SignSandbox/TeamInvitesDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TeamInvitesDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        try
        {
            var response = new TeamApi(config).TeamInvites();

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Team#TeamInvites: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
